### ISSUE TITLE
The cookie message alert does not have good enough colour contrast

### ISSUE THREAD

**Issue by a18osken** [HGustavs/LenaSYS#9822]
In order to pass the [lighthouse](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk?hl=sv) audit, the colour contrast of the cookie message needs to be changed while still meshing well with LenaSYS's color palette.
This current contrast is insufficient
![CookieContrast](https://user-images.githubusercontent.com/62884391/113292991-828f3c00-92f5-11eb-9a51-11b435245fdd.PNG)
 
Part of issue #9800

**Comment by a18osken** [comment]
The colour scheme for LenaSYS may be viewed [here](https://github.com/HGustavs/LenaSYS/wiki/CodeViewer)

**Comment by HGustavs** [comment]
Nice catch.

I would say making the text darker is the best option rather than making the background lighter.

Can we make a mock-up of what it would look like when fixed?

**Comment by a18osken** [comment]
I was considering increasing the hue and darkness of the foreground color to look more like this
![CookieContrast_Rev2](https://user-images.githubusercontent.com/62884391/113829204-81429100-9785-11eb-8b38-fd496d43edc5.PNG)
This would likely pass most WCAG 2.0 audits.
